### DISCUSSION TITLE
custom notifications on discord

### DISCUSSION THREAD

**Discussion by lipefur** [swar/Swar-Chia-Plot-Manager#548] [General]
I am trying to configure the discord notification to notify the time elapsed in the completion of each plot.

how could I integrate elapsed_time into your body

**log.py**
```            send_notifications(
                title='Plot Completed',
                body=f'You completed a plot on {socket.gethostname()}!',
                settings=notification_settings,
            )```

**Comment by Lyushen** [comment]
Same thoughts, pretty interested in details, not only the time, task name, phases time and errors if there any
